app/src/androidTest/java/com/sdp/movemeet/Profile/EditProfileActivityTest.java

```diff
@@ -42,12 +45,6 @@ public void updateProfileByEnteringFieldsAndClickingSaveButton() {
         onView(ViewMatchers.withId(R.id.edit_text_edit_profile_description))
                 .perform(replaceText(TEST_DESCRIPTION), closeSoftKeyboard());
 
-        try {
-            Thread.sleep(1000);
```

Comment:
Why did you remove the Thread.sleep?

Reply:
It was a priori useless, the tests passed anyway

Reply:
Nice